### PR TITLE
Mega Rename

### PR DESCRIPTION
**Pull request by anstaf**
todo

### CODE REVIEW

**Comment by anstaf** [comment]
launch jenkins

**Comment by anstaf** [comment]
launch jenkins

**Comment by anstaf** [comment]
launch jenkins

**Comment by anstaf** [comment]
launch jenkins

**Comment by anstaf** [comment]
launch jenkins

**Comment by anstaf** [comment]
launch jenkins

**Comment by anstaf** [comment]
launch jenkins

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -20,21 +20,23 @@ function(install_example)
 endfunction()
 
 if(GT_INSTALL_EXAMPLES)
+    message(STATUS "fuck!!! ${GT_INSTALL_EXAMPLES_PATH}")
```
Comment (havogt) [inline examples/CMakeLists.txt:23]: Is this a check if I do a proper review?

Comment (anstaf) [reply examples/CMakeLists.txt:23]: opps. debugging leftover. will fix now

**Comment by anstaf** [comment]
launch jenkins

**Pull request merged**